docs/hooks.rst

```diff
@@ -138,7 +138,7 @@ You may actually drop validation exception inside save, insert or update hooks::
 
     $m->onHook(Model::HOOK_BEFORE_SAVE, function($m) {
         if ($m->get('name') === 'Yagi') {
-            throw new \atk4\data\ValidationException('We don't serve like you');
+            throw new \atk4\data\ValidationException(['name' => "We don't serve like you"]);
```

Comment:
Good catch, I forget to push this. Done these changes in 3 copies of this repo... :)